pkg/sidecar/sidecar.go

```diff
@@ -7,11 +7,13 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/ipfs/testground/pkg/logging"
 	"github.com/ipfs/testground/sdk/sync"
 )
 
 var runners = map[string]func() (InstanceManager, error){
-	"docker": NewDockerManager,
+	//"docker": NewDockerManager,
```

Comment:
?

Reply:
Will revert. I had the wrong image installed, and before realising it was not clear whether the problem was the code or the wrong docker tag/image.